display.c

```diff
@@ -192,104 +190,40 @@ extern int display_keyaction(struct mtr_ctl *ctl)
 
 extern void display_rawxmit(struct mtr_ctl *ctl, int host, int seq)
 {
-  switch(ctl->DisplayMode) {
-  case DisplayRaw:
+  if (ctl->DisplayMode == DisplayRaw)
     raw_rawxmit (host, seq);
-    break;
-  }
 }
 
 
 extern void display_rawping(struct mtr_ctl *ctl, int host, int msec, int seq)
 {
-  switch(ctl->DisplayMode) {
-  case DisplayReport:
-  case DisplayTXT:
-  case DisplayJSON:
-  case DisplayXML:
-  case DisplayCSV:
-  case DisplaySplit:
-#ifdef HAVE_NCURSES
-  case DisplayCurses:
-#endif
-#ifdef HAVE_GTK
-  case DisplayGTK:
-#endif
-    break;
-  case DisplayRaw:
+  if (ctl->DisplayMode == DisplayRaw)
     raw_rawping (ctl, host, msec, seq);
-    break;
-  }
 }
 
 
 extern void display_rawhost(struct mtr_ctl *ctl, int host, ip_t *ip_addr)
 {
-  switch(ctl->DisplayMode) {
-  case DisplayReport:
-  case DisplayTXT:
-  case DisplayJSON:
-  case DisplayXML:
-  case DisplayCSV:
-  case DisplaySplit:
-#ifdef HAVE_NCURSES
-  case DisplayCurses:
-#endif
-#ifdef HAVE_GTK
-  case DisplayGTK:
-#endif
-    break;
-  case DisplayRaw:
+  if (ctl->DisplayMode == DisplayRaw)
     raw_rawhost (ctl, host, ip_addr);
-    break;
-  }
 }
 
 
 extern void display_loop(struct mtr_ctl *ctl)
 {
```

Comment:
In deed. And earlier code could have been cleaned up using default, but this is now as it is. If you want I can convert back to switch case + default. Downside is few lines more, and quite frankly this is just a style question and maintainers opinion is the law. Just tell if change is needed.